tests/ecmp/test_inner_hashing.py

```diff
@@ -35,6 +35,46 @@
 VXLAN_PORT = 13330
 DUT_VXLAN_PORT_JSON_FILE = '/tmp/vxlan.switch.json'
 
+IP_VERSIONS_LIST = ["ipv4", "ipv6"]
+TABLE_NAME = "pbh_table"
+TABLE_DESCRIPTION = "NVGRE and VXLAN"
+HASH_NAME = "inner_hash"
+VXLAN_RULE_NAME = "vxlan_{}_{}"
+NVGRE_RULE_NAME = "nvgre_{}_{}"
+VXLAN_RULE_PRIO = "1"
+NVGRE_RULE_PRIO = "2"
+ECMP_PACKET_ACTION = "SET_ECMP_HASH"
+V4_ETHER_TYPE = "0x0800"
+V6_ETHER_TYPE = "0x86dd"
+VXLAN_IP_PROTOCOL = "0x11"
+NVGRE_IP_PROTOCOL = "0x2f"
+VXLAN_L4_DST_PORT = "0x3412"
+VXLAN_L4_DST_PORT_OPTION = " --l4-dst-port {}".format(VXLAN_L4_DST_PORT)
+ADD_PBH_TABLE_CMD = "sudo config pbh table add '{}' --interface-list '{}' --description '{}'"
+DEL_PBH_TABLE_CMD = "sudo config pbh table delete '{}'"
+ADD_PBH_RULE_BASE_CMD = "sudo config pbh rule add '{}' '{}' --priority '{}' --ether-type {}" \
+                        " --inner-ether-type '{}' --hash '{}' --packet-action '{}' --flow-counter 'ENABLED'"
+DEL_PBH_RULE_CMD = "sudo config pbh rule delete '{}' '{}'"
+ADD_PBH_HASH_CMD = "sudo config pbh hash add '{}' --hash-field-list '{}'"
+DEL_PBH_HASH_CMD = "sudo config pbh hash delete '{}'"
+ADD_PBH_HASH_FIELD_CMD = "sudo config pbh hash-field add '{}' --hash-field '{}' --sequence-id '{}'"
+DEL_PBH_HASH_FIELD_CMD = "sudo config pbh hash-field delete '{}'"
+
+PBH_HASH_FIELD_LIST = "inner_ip_proto," \
+                      "inner_l4_dst_port,inner_l4_src_port," \
+                      "inner_dst_ipv4,inner_src_ipv4," \
+                      "inner_src_ipv6,inner_dst_ipv6"
+HASH_FIELD_CONFIG = {
+    "inner_ip_proto": {"field": "INNER_IP_PROTOCOL", "sequence": "1"},
+    "inner_l4_dst_port": {"field": "INNER_L4_DST_PORT", "sequence": "2"},
+    "inner_l4_src_port": {"field": "INNER_L4_SRC_PORT", "sequence": "2"},
+    "inner_src_ipv4": {"field": "INNER_SRC_IPV4", "sequence": "3", "mask": "255.255.255.255"},
+    "inner_dst_ipv4": {"field": "INNER_DST_IPV4", "sequence": "3", "mask": "255.255.255.255"},
+    "inner_src_ipv6": {"field": "INNER_SRC_IPV6", "sequence": "4", "mask": "ffff:ffff::"},
+    "inner_dst_ipv6": {"field": "INNER_DST_IPV6", "sequence": "4", "mask": "ffff:ffff::"}
```

Comment:
@AntonHryshchuk this mask doesn't fit the IPv6 address range, so load balancing won't be working:
```python
SRC_IPV6_RANGE = ['20D0:A800:0:00::', '20D0:A800:0:00::FFFF']
DST_IPV6_RANGE = ['20D0:A800:0:01::', '20D0:A800:0:01::FFFF']
```
Please fix the test logic.

Reply:
the ipv6 mask changed to "::ffff:ffff"

Reply:
@anish-n please have a look